pkg/simple/client/es/client.go

```diff
@@ -34,9 +36,10 @@ import (
 )
 
 const (
-	ElasticV5 = "5"
-	ElasticV6 = "6"
-	ElasticV7 = "7"
+	ElasticV5    = "5"
+	ElasticV6    = "6"
+	ElasticV7    = "7"
+	OpenSearchV2 = "2"
```

Comment:
The version can be changed like below to distinguish it with elasticsearch versions
```suggestion
	OpenSearchV2 = "OpenSearchV2"
```

Reply:
We determine the version by the `version` field in the data returned in the response, which, if set up this way, is not well handled on the back end.
https://github.com/kubesphere/kubesphere/blob/master/pkg/simple/client/es/client.go#L125

Reply:
Is there some field returned in the response that contains `opensearch`?

Reply:
This field is returned, so this constant I think is better set to `2`
![1657367628376](https://user-images.githubusercontent.com/39892300/178104525-949573a0-b2a4-4d37-807a-e8c9244e2a0f.jpg)

Reply:
I mean the distribution field contains "opensearch"
So if this field is "opensearch", we can add a `opensearchv` prefix to the version 2 and that will make the version become `opensearchv2`

Reply:
![a5b793b04e2efd33257844887610184](https://user-images.githubusercontent.com/39892300/178127506-4812b04e-058b-42d7-bc06-e3a73cba82a5.png)
I tested the return fields of elasticsearch and there is no `distribution` field in es, which should be the difference between elasticsearch and OpenSearch.

Reply:
So when there is distribution field and the value is opensearch, we can add `opensearchv` prefix to the v